Experiments/Experiments/FeatureFlag.swift

```diff
@@ -65,4 +65,11 @@ public enum FeatureFlag: Int {
     /// Enable optimistic updates for orders
     ///
     case updateOrderOptimistically
+
+    /// Enable optimistic updates for order address if are enabled.
+    ///
+    /// This feature flag will be removed when development to enable
+    /// optimistic updating of the address of an order is finished.
+    ///
+    case useUpdateOrderAddressOptimisticallyIfAvailable
```

Comment:
This is the new feature flag. Do you think it's not needed? 🤔 

BTW, this feature flag will be removed in the part 3 😅

Reply:
Yep, I agree with you that we could use `updateOrderOptimistically` 👍🏼 I made the change [here](https://github.com/woocommerce/woocommerce-ios/pull/6724/commits/d5a9149c43cdcce8ac87099b77407218478d8146) . 

Thanks for the clarification 🙂